lib/msal-common/src/auth/MsalAccount.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ClientInfo } from "./ClientInfo";
+import { IdToken } from "./IdToken";
+import { StringUtils } from "../utils/StringUtils";
+import { StringDict } from "../utils/MsalTypes";
+import { ICrypto } from "../utils/crypto/ICrypto";
+
+/**
+ * accountIdentifier       combination of idToken.uid and idToken.utid
+ * homeAccountIdentifier   combination of clientInfo.uid and clientInfo.utid
+ * userName                idToken.preferred_username
+ * name                    idToken.name
+ * idToken                 idToken
+ * sid                     idToken.sid - session identifier
+ * environment             idtoken.issuer (the authority that issues the token)
+ */
+export class MsalAccount {
+
+    accountIdentifier: string;
+    homeAccountIdentifier: string;
+    userName: string;
+    name: string;
+    idToken: StringDict; // will be deprecated soon
```

Comment:
This is the chance to do the deprecation, so I think this should be removed.

Reply:
Yes, I will change this to string in future PRs.